pkg/server/server.go

```diff
@@ -9,21 +9,23 @@ import (
 
 	"github.com/golang/glog"
 
-	"github.com/pmorie/go-open-service-broker-skeleton/pkg/broker"
+	"github.com/pmorie/go-open-service-broker-skeleton/pkg/rest"
 )
 
-type server struct {
-	api broker.APISurface
+// Server might be redundant :)
```

Comment:
Yeah I agree, took the scaffolding code from me UPS broker.

Reply:
Yeah, that was a note to myself to collapse them but I ran out of time.